pom.xml

```diff
@@ -344,5 +344,10 @@
       <artifactId>graphql-java</artifactId>
       <version>4.2</version>
     </dependency>
+      <dependency>
+          <groupId>com.google.code.gson</groupId>
+          <artifactId>gson</artifactId>
```

Comment:
Gson (to me) seems to have simpler interfaces for handling json.  I don't know if we should make the switch.  And I'm not sure we need the GeoJSON support any longer (because pattern shapes are now lists of shape points).

Reply:
Can you give examples of these differences? Jackson has a streaming JSON API that I've found handy in several places to avoid loading huge amounts of data into memory, but I don't really know what the alternative looks like for basic serialization/deserialization actions.